### DISCUSSION TITLE
Thunderburd VJOURNAL support

### DISCUSSION THREAD

**Discussion by woifes** [TechbeeAT/jtxBoard#1088] [Show and tell]
Since we all use and love jtx board for multiple things we know that there is missing a desktop app though. For contacts, calendar and todos I use Thunderbird. To request the support for Journals aswell I submitted a Idea in Thunderbird. If you like you can support my [Idea](https://connect.mozilla.org/t5/ideas/support-vjournal-in-thunderbird/idi-p/46295)

**Comment by patrickunterwegs** [comment]
Thanks @woifes ! I'll also post on Mastodon to find more people to endorse your post! 😊

**Comment by Julianoe** [comment]
@woifes good idea! I supported your request and will share it online
